populate_db.py

```diff
@@ -0,0 +1,107 @@
+import pyphen, requests, json
+#pyphen.language_fallback('ipa_en')
+ipa_dic = pyphen.Pyphen(lang='ipa_en')
+spelling_dic = pyphen.Pyphen(lang='en')
+
+URL = "http://127.0.0.1:5000"
+LANGUAGE = "english"
+
+
+print(ipa_dic.inserted("sʌbdʒɛktɪv"))  # use dark magic space trick in the future
+
+wordlist = "allprons.txt"
+data = open(wordlist, mode="r", encoding="utf8").read()
+
+separators = data.splitlines()[0].split()
+
+raw_data = data.splitlines()[1:]
```

Comment:
same here. plus you are calling `splitlines` twice. better call it once, check if it has one element to get the separators, and check if it has more than 2 to get the raw_data